<a href="https://colab.research.google.com/github/JoaoSchneiderr/Minera-o-de-Dados/blob/main/notebooks/Getting_started_with_google_colab_ai.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

### Importação de informações

In [2]:
!pip install gdown
!gdown --id 16dDk7X88HATXyy9UqLwapK4YgMxVn8ak -O dados_nova_iguacu_limpo_v3.csv
import pandas as pd

/usr/local/lib/python3.12/dist-packages/gdown/__main__.py:139: FutureWarning: Option `--id` was deprecated in version 4.3.1 and will be removed in 5.0. You don't need to pass it anymore to use a file ID.
  warnings.warn(
Downloading...
From: https://drive.google.com/uc?id=16dDk7X88HATXyy9UqLwapK4YgMxVn8ak
To: /content/dados_nova_iguacu_limpo_v3.csv
100% 359k/359k [00:00<00:00, 97.3MB/s]


In [3]:
nova_iguacu_df = pd.read_csv('dados_nova_iguacu_limpo_v3.csv', encoding='latin1', on_bad_lines='skip')
display(nova_iguacu_df.head())

,id,pesid,data_inversa,dia_semana,horario,uf,br,km,municipio,causa_principal,...,idade,sexo,ilesos,feridos_leves,feridos_graves,mortos,latitude,longitude,faixa_etaria,Unnamed: 35
0,653257.0,1469809.0,2025-01-04,sábado,23:00:00,RJ,493.0,66,NOVA IGUACU,Não,...,53.0,Masculino,1.0,0.0,0.0,0,-43.453331,Adulto,NaN,NaN
1,653257.0,1469811.0,2025-01-04,sábado,23:00:00,RJ,493.0,66,NOVA IGUACU,Não,...,33.0,Masculino,0.0,0.0,1.0,1,-43.453331,Adulto,NaN,NaN
2,653414.0,1470243.0,2025-01-05,domingo,22:50:00,RJ,116.0,181,NOVA IGUACU,Sim,...,47.0,Masculino,1.0,0.0,0.0,0,-22.761312,-43.423492,Adulto,NaN
3,654387.0,1472899.0,2025-01-10,sexta-feira,16:30:00,RJ,493.0,73,NOVA IGUACU,Sim,...,51.0,Masculino,1.0,0.0,0.0,0,-22.662866,-43.6250497,Adulto,NaN
4,654568.0,1473371.0,2025-01-12,domingo,01:00:00,RJ,465.0,18,NOVA IGUACU,Sim,...,23.0,Masculino,0.0,1.0,0.0,0,-22.831354,-43.615128,Jovem,NaN


### 1. Quantidade total de acidentes em Nova Iguaçu / Quantidade de acidentes registrados por mês


In [ ]:
acidentes_total = len(nova_iguacu_df)
acidentes_total

1081

In [ ]:
acidentes_por_mes = nova_iguacu_df.copy()
acidentes_por_mes['data_inversa'] = pd.to_datetime(acidentes_por_mes['data_inversa'])

meses_pt = {
    1: 'Janeiro', 2: 'Fevereiro', 3: 'Março', 4: 'Abril',
    5: 'Maio', 6: 'Junho', 7: 'Julho', 8: 'Agosto',
    9: 'Setembro', 10: 'Outubro', 11: 'Novembro', 12: 'Dezembro'
}

contagem_mes = acidentes_por_mes['data_inversa'].dt.month.value_counts().sort_index()
contagem_mes.index.name = 'Divisão por mês'
display(contagem_mes.rename(index=meses_pt))

,count
Divisão por mês,
Janeiro,74
Fevereiro,80
Março,98
Abril,113
Maio,81
Junho,95
Julho,81
Agosto,78
Setembro,82


### 2. Gravidade dos acidentes por mês em Nova Iguaçu


In [ ]:
nova_iguacu_df['data_inversa'] = pd.to_datetime(nova_iguacu_df['data_inversa'])
nova_iguacu_df['mês'] = nova_iguacu_df['data_inversa'].dt.month

gravidade_mensal = nova_iguacu_df.groupby('mês')[['mortos', 'feridos_graves', 'feridos_leves', 'ilesos']].sum()

print("\nAnálise da Gravidade dos Acidentes por Mês em Nova Iguaçu (Média Mensal):")
avg_mortos_mes = gravidade_mensal['mortos'].mean()
avg_feridos_graves_mes = gravidade_mensal['feridos_graves'].mean()
avg_feridos_leves_mes = gravidade_mensal['feridos_leves'].mean()
print(f"Em média, mensalmente, ocorreram {avg_mortos_mes:.1f} mortes, {avg_feridos_graves_mes:.1f} feridos graves e {avg_feridos_leves_mes:.1f} feridos leves.")

total_mortos_geral = gravidade_mensal['mortos'].sum()
total_feridos_graves_geral = gravidade_mensal['feridos_graves'].sum()
total_feridos_leves_geral = gravidade_mensal['feridos_leves'].sum()
total_ilesos_geral = gravidade_mensal['ilesos'].sum()

print("\nTotais Gerais e Proporções (Ano de 2025) ")
total_mortos = total_mortos_geral
total_feridos = total_feridos_graves_geral + total_feridos_leves_geral
print(f"Total Geral de Mortos: {total_mortos:.0f}")
print(f"Total Geral de Feridos (Graves + Leves): {total_feridos:.0f}")


Análise da Gravidade dos Acidentes por Mês em Nova Iguaçu (Média Mensal):
Em média, mensalmente, ocorreram 1.4 mortes, 5.6 feridos graves e 34.9 feridos leves.

Totais Gerais e Proporções (Ano de 2025) 
Total Geral de Mortos: 17
Total Geral de Feridos (Graves + Leves): 486


In [ ]:
classificacao_acidente_contagem = nova_iguacu_df['classificacao_acidente'].value_counts()

acidentes_com_vitimas_fatal = classificacao_acidente_contagem.get('Com Vítimas Fatais', 0)
acidentes_com_vitimas_feridas = classificacao_acidente_contagem.get('Com Vítimas Feridas', 0)
acidentes_sem_vitimas = classificacao_acidente_contagem.get('Sem Vítimas', 0)

total_registros_acidentes = len(nova_iguacu_df)

prop_com_vitimas = ((acidentes_com_vitimas_fatal + acidentes_com_vitimas_feridas) / total_registros_acidentes) * 100
prop_sem_vitimas = (acidentes_sem_vitimas / total_registros_acidentes) * 100

print(f"Proporção de registros de acidentes com vítimas (fatais ou feridas): {prop_com_vitimas:.2f}%")
print(f"Proporção de registros de acidentes sem vítimas: {prop_sem_vitimas:.2f}%")

Proporção de registros de acidentes com vítimas (fatais ou feridas): 89.82%
Proporção de registros de acidentes sem vítimas: 10.08%


In [ ]:
taxa_acidentes_morte = (acidentes_com_vitimas_fatal / total_registros_acidentes) * 100
print(f"Taxa de registros de acidentes com vítimas fatais: {taxa_acidentes_morte:.2f}%")

Taxa de registros de acidentes com vítimas fatais: 4.90%


In [ ]:
taxa_acidentes_morte = (acidentes_com_vitimas_fatal / total_registros_acidentes) * 100
print(f"Taxa de registros de acidentes com vítimas fatais: {taxa_acidentes_morte:.2f}%")


total_unique_accidents = nova_iguacu_df['id'].nunique()
total_vitimas = total_mortos_geral + total_feridos_graves_geral + total_feridos_leves_geral
media_vitimas_por_acidente = total_vitimas / total_unique_accidents
print(f"Média de vítimas (mortos + feridos graves + feridos leves) por acidente único: {media_vitimas_por_acidente:.2f}")

Taxa de registros de acidentes com vítimas fatais: 4.90%
Média de vítimas (mortos + feridos graves + feridos leves) por acidente único: 1.15


### 3. Tipos de acidentes


In [ ]:
print("\n Tipos de Acidentes Mais Comuns")
tipos_acidentes_comuns = nova_iguacu_df['tipo_acidente'].value_counts().head(10)
display(tipos_acidentes_comuns)


 Tipos de Acidentes Mais Comuns


,count
tipo_acidente,
Colisão traseira,301
Colisão lateral mesmo sentido,203
Queda de ocupante de veículo,198
Colisão transversal,64
Atropelamento de Pedestre,54
Tombamento,51
Engavetamento,49
Colisão com objeto,39
Saída de leito carroçável,35


In [ ]:
print("\n Classificação de Acidentes Mais Comuns")
classificacao_acidente_comum = nova_iguacu_df['classificacao_acidente'].value_counts()
display(classificacao_acidente_comum)


 Classificação de Acidentes Mais Comuns


,count
classificacao_acidente,
Com Vítimas Feridas,918
Sem Vítimas,109
Com Vítimas Fatais,53


### 4. Veículos envolvidos


In [ ]:
print("\n Top 10 Tipos de Veículos Mais Comuns nos Acidentes em Nova Iguaçu ")
tipos_veiculo_comuns = nova_iguacu_df['tipo_veiculo'].value_counts().head(10)
display(tipos_veiculo_comuns)


 Top 10 Tipos de Veículos Mais Comuns nos Acidentes em Nova Iguaçu 


,count
tipo_veiculo,
Motocicleta,414
Automóvel,380
Caminhonete,55
Caminhão,41
Semireboque,39
Camioneta,34
Caminhão-trator,27
Desconhecido,27
Motoneta,16


In [ ]:
print("\n CARRO VS MOTO VS CAMINHÃO")
veiculo_especifico = nova_iguacu_df['tipo_veiculo'].value_counts().loc[['Automóvel', 'Motocicleta', 'Caminhão']]
display(veiculo_especifico)


 CARRO VS MOTO VS CAMINHÃO


,count
tipo_veiculo,
Automóvel,380
Motocicleta,414
Caminhão,41


### 5. Perfil das pessoas envolvidas


In [ ]:
df_perfil = nova_iguacu_df[['sexo', 'idade', 'estado_fisico', 'tipo_envolvido']].copy()
df_perfil['idade'] = pd.to_numeric(df_perfil['idade'], errors='coerce')
df_perfil = df_perfil.dropna(subset=['idade'])
display(df_perfil.head())

,sexo,idade,estado_fisico,tipo_envolvido
0,Masculino,53.0,Ileso,Condutor
1,Masculino,33.0,Óbito,Pedestre
2,Masculino,47.0,Ileso,Condutor
3,Masculino,51.0,Ileso,Condutor
4,Masculino,23.0,Lesões Leves,Condutor


In [ ]:
bins = [0, 18, 25, 35, 45, 55, 65, 100]
labels = ['0-17', '18-24', '25-34', '35-44', '45-54', '55-64', '65+']

df_perfil['faixa_etaria'] = pd.cut(df_perfil['idade'], bins=bins, labels=labels, right=False)

print("\nDistribuição por Faixa Etária (Indivíduos Envolvidos em Acidentes):")
display(df_perfil['faixa_etaria'].value_counts().sort_index())


Distribuição por Faixa Etária (Indivíduos Envolvidos em Acidentes):


,count
faixa_etaria,
0-17,16
18-24,112
25-34,282
35-44,245
45-54,156
55-64,88
65+,41


In [ ]:
df_vitimas = df_perfil[df_perfil['estado_fisico'].isin(['Ferido Grave', 'Ferido Leve', 'Óbito'])]
idade_media_vitimas = df_vitimas['idade'].mean()

print(f"Idade média das vítimas (Ferido Grave, Ferido Leve ou Morto): {idade_media_vitimas:.2f} anos")

Idade média das vítimas (Ferido Grave, Ferido Leve ou Morto): 26.35 anos


In [ ]:
print("\nDistribuição por Sexo (Indivíduos Envolvidos em Acidentes):")
contagem_sexo = df_perfil['sexo'].value_counts()
display(contagem_sexo)

porcentagem_sexo = df_perfil['sexo'].value_counts(normalize=True) * 100
print("\nPorcentagem por Sexo (Indivíduos Envolvidos em Acidentes):")
display(porcentagem_sexo)


Distribuição por Sexo (Indivíduos Envolvidos em Acidentes):


,count
sexo,
Masculino,778
Feminino,192
Não Informado,107
Ignorado,3



Porcentagem por Sexo (Indivíduos Envolvidos em Acidentes):


,proportion
sexo,
Masculino,72.037037
Feminino,17.777778
Não Informado,9.907407
Ignorado,0.277778


In [ ]:
print("\nContagem de Condutores vs. Passageiros:\n")
contagem_condutor_passageiro = df_perfil['tipo_envolvido'].value_counts().loc[['Condutor', 'Passageiro']]
display(contagem_condutor_passageiro)


Contagem de Condutores vs. Passageiros:



,count
tipo_envolvido,
Condutor,797
Passageiro,210


### 6. Distribuição temporal


In [ ]:
print("\n Acidentes por Horário em Nova Iguaçu")
acidentes_por_horario = nova_iguacu_df['horario'].value_counts().sort_values(ascending=False)
display(acidentes_por_horario)



 Acidentes por Horário em Nova Iguaçu


,count
horario,
08:00:00,32
20:30:00,21
19:00:00,20
20:00:00,20
07:00:00,19
...,...
21:50:00,1
04:40:00,1
06:47:00,1


In [ ]:
print("\nAcidentes por Dia da Semana em Nova Iguaçu")
acidentes_por_dia_semana = nova_iguacu_df['dia_semana'].value_counts().sort_values(ascending=False)
display(acidentes_por_dia_semana)


Acidentes por Dia da Semana em Nova Iguaçu


,count
dia_semana,
segunda-feira,188
domingo,181
sexta-feira,166
terça-feira,160
quarta-feira,153
sábado,147
quinta-feira,85


In [ ]:
print("\n Acidentes por Condição Meteorológica em Nova Iguaçu")
acidentes_por_condicao_metereologica = nova_iguacu_df['condicao_metereologica'].value_counts().sort_values(ascending=False)
display(acidentes_por_condicao_metereologica)


 Acidentes por Condição Meteorológica em Nova Iguaçu


,count
condicao_metereologica,
Céu Claro,790
Nublado,121
Chuva,58
Sol,51
Garoa/Chuvisco,41
Ignorado,19


In [ ]:
print("\n Acidentes por Fase do Dia em Nova Iguaçu ")
acidentes_por_fase_dia = nova_iguacu_df['fase_dia'].value_counts()
display(acidentes_por_fase_dia)


 Acidentes por Fase do Dia em Nova Iguaçu 


,count
fase_dia,
Pleno dia,652
Plena Noite,334
Amanhecer,49
Anoitecer,45


In [ ]:
print("\n Gravidade dos Acidentes por Fase do Dia em Nova Iguaçu ")

# Convert relevant columns to numeric, coercing errors to NaN
nova_iguacu_df['mortos'] = pd.to_numeric(nova_iguacu_df['mortos'], errors='coerce')
nova_iguacu_df['feridos_graves'] = pd.to_numeric(nova_iguacu_df['feridos_graves'], errors='coerce')
nova_iguacu_df['feridos_leves'] = pd.to_numeric(nova_iguacu_df['feridos_leves'], errors='coerce')

gravidade_por_fase_dia = nova_iguacu_df.groupby('fase_dia')[['mortos', 'feridos_graves', 'feridos_leves']].sum()
display(gravidade_por_fase_dia)


 Gravidade dos Acidentes por Fase do Dia em Nova Iguaçu 


,mortos,feridos_graves,feridos_leves
fase_dia,,,
Amanhecer,2.0,4.0,14.0
Anoitecer,0.0,2.0,21.0
Plena Noite,11.0,27.0,128.0
Pleno dia,4.0,34.0,256.0


### 7. Condições do ambiente



In [ ]:
print("\n Acidentes por Tipo de Pista em Nova Iguaçu ")
acidentes_por_tipo_pista = nova_iguacu_df['tipo_pista'].value_counts()
display(acidentes_por_tipo_pista)


 Acidentes por Tipo de Pista em Nova Iguaçu 


,count
tipo_pista,
Dupla,694
Simples,216
Múltipla,170


In [ ]:
print("\n Acidentes por Uso do Solo em Nova Iguaçu")
acidentes_por_uso_solo = nova_iguacu_df['uso_solo'].value_counts()
display(acidentes_por_uso_solo)


 Acidentes por Uso do Solo em Nova Iguaçu


,count
uso_solo,
Sim,714
Não,366


In [ ]:
print("\n Acidentes por Condição Meteorológica em Nova Iguaçu")
acidentes_por_condicao_metereologica = nova_iguacu_df['condicao_metereologica'].value_counts()
display(acidentes_por_condicao_metereologica)


 Acidentes por Condição Meteorológica em Nova Iguaçu


,count
condicao_metereologica,
Céu Claro,790
Nublado,121
Chuva,58
Sol,51
Garoa/Chuvisco,41
Ignorado,19


In [ ]:
print("\n Acidentes por Traçado da Via em Nova Iguaçu ")
acidentes_por_tracado_via = nova_iguacu_df['tracado_via'].value_counts()
display(acidentes_por_tracado_via)


 Acidentes por Traçado da Via em Nova Iguaçu 


,count
tracado_via,
Reta,981
Curva,43
Reta;Em Obras,29
Em Obras;Reta,4
Aclive;Reta,4
Reta;Interseção de Vias,3
Interseção de Vias,3
Reta;Aclive,3
Interseção de Vias;Curva,3


### 8. Localização dos acidentes

In [ ]:
print("\n Acidentes por Rodovia (BR) em Nova Iguaçu ")
acidentes_por_br = nova_iguacu_df['br'].value_counts()
display(acidentes_por_br)


 Acidentes por Rodovia (BR) em Nova Iguaçu 


,count
br,
116.0,778
465.0,185
493.0,113
40.0,1


In [ ]:
df_br_km_acidentes = acidentes_por_br_km.reset_index()
df_br_km_acidentes.columns = ['br_km_', 'ACIDENTES']

df_br_km_acidentes[['BR', 'KM']] = df_br_km_acidentes['br_km_'].str.split('-', expand=True)

df_br_km_acidentes = df_br_km_acidentes[['BR', 'KM', 'ACIDENTES']]
df_br_km_acidentes.index = df_br_km_acidentes.index + 1

print("\nContagem de Acidentes por BR e KM:")
display(df_br_km_acidentes)


Contagem de Acidentes por BR e KM:


,BR,KM,ACIDENTES
1,116.0,186,111
2,116.0,184,92
3,116.0,183,79
4,116.0,185,73
5,116.0,181,71
6,116.0,187,62
7,465.0,20,59
8,116.0,182,55
9,465.0,21,52
10,116.0,188,45


In [ ]:
print("\n Análise de Pontos Críticos por Latitude e Longitude (Top 10 mais frequentes)")
nova_iguacu_df['lat_lon'] = nova_iguacu_df['latitude'].astype(str) + ',' + nova_iguacu_df['longitude'].astype(str)
acidentes_por_lat_lon = nova_iguacu_df['lat_lon'].value_counts().head(10)
display(acidentes_por_lat_lon)


 Análise de Pontos Críticos por Latitude e Longitude (Top 10 mais frequentes)


,count
lat_lon,
"-22.73962339,-43.46276495",30
"-22.73883756,-43.47245197",19
"-22.68140623,-43.53034051",14
"-22.74151051,-43.45351823",11
"-22.7380709,-43.48213999",10
"-22.846828,-43.605339",10
"-22.74185146,-43.52263989",8
"-22.84934285,-43.603184",8
"-22.688636,-43.439037",7


### 9. Análise do Acidente Mais Grave







In [25]:
acidentes_analise = (
    nova_iguacu_df.groupby('id')
    .agg(
        total_mortos=('mortos', 'sum'),
        total_feridos_graves=('feridos_graves', 'sum'),
        total_feridos_leves=('feridos_leves', 'sum'),
        total_veiculos_envolvidos=('tipo_veiculo', 'count')
    )
    .reset_index()
)
acidentes_analise['total_feridos'] = (
    acidentes_analise['total_feridos_graves'] +
    acidentes_analise['total_feridos_leves']
)

print("\nInformações Principais dos Acidentes Mais Graves:")

acidente_mais_mortos_id = acidentes_analise[acidentes_analise['total_mortos'] == acidentes_analise['total_mortos'].max()]['id'].iloc[0]

for acc_id in [int(acidente_mais_mortos_id)]:
    print(f"\nResumo do Acidente ID: {acc_id}")

    agg_info = acidentes_analise[acidentes_analise['id'] == acc_id].iloc[0]
    print(f"Número de mortos: {int(agg_info['total_mortos'])}")
    print(f"Número de feridos (graves + leves): {int(agg_info['total_feridos'])}")
    print(f"Número de envolvidos (registros de pessoas/veículos): {int(agg_info['total_veiculos_envolvidos'])}")

    detail_info = nova_iguacu_df[nova_iguacu_df['id'] == acc_id].iloc[0]
    print(f"BR: {detail_info['br']}")
    print(f"KM: {detail_info['km']}")
    print(f"Dia da Semana: {detail_info['dia_semana']}")
    print(f"Horário: {detail_info['horario']}")
    print(f"Causa do Acidente: {detail_info['causa_acidente']}")

    vehicle_types = nova_iguacu_df[nova_iguacu_df['id'] == acc_id]['tipo_veiculo'].unique()
    print(f"Tipos de Veículos Envolvidos: {', '.join(vehicle_types)}")


Informações Principais dos Acidentes Mais Graves:

Resumo do Acidente ID: 687006
Número de mortos: 101
Número de feridos (graves + leves): 0
Número de envolvidos (registros de pessoas/veículos): 3
BR: 493.0
KM: 59
Dia da Semana: quarta-feira
Horário: 20:00:00
Causa do Acidente: Pedestre andava na pista
Tipos de Veículos Envolvidos: Bicicleta, Automóvel, Desconhecido


In [24]:
acidentes_analise = (
    nova_iguacu_df.groupby('id')
    .agg(
        total_mortos=('mortos', 'sum'),
        total_feridos_graves=('feridos_graves', 'sum'),
        total_feridos_leves=('feridos_leves', 'sum'),
        total_veiculos_envolvidos=('tipo_veiculo', 'count')
    )
    .reset_index()
)
acidentes_analise['total_feridos'] = (
    acidentes_analise['total_feridos_graves'] +
    acidentes_analise['total_feridos_leves']
)

print("\nInformações Principais dos Acidentes Mais Graves:")

acidente_mais_feridos_id = acidentes_analise[acidentes_analise['total_feridos'] == acidentes_analise['total_feridos'].max()]['id'].iloc[0]

for acc_id in [int(acidente_mais_feridos_id)]:
    print(f"\nResumo do Acidente ID: {acc_id}")

    agg_info = acidentes_analise[acidentes_analise['id'] == acc_id].iloc[0]
    print(f"Número de mortos: {int(agg_info['total_mortos'])}")
    print(f"Número de feridos (graves + leves): {int(agg_info['total_feridos'])}")
    print(f"Número de envolvidos (registros de pessoas/veículos): {int(agg_info['total_veiculos_envolvidos'])}")

    detail_info = nova_iguacu_df[nova_iguacu_df['id'] == acc_id].iloc[0]
    print(f"BR: {detail_info['br']}")
    print(f"KM: {detail_info['km']}")
    print(f"Dia da Semana: {detail_info['dia_semana']}")
    print(f"Horário: {detail_info['horario']}")
    print(f"Causa do Acidente: {detail_info['causa_acidente']}")

    vehicle_types = nova_iguacu_df[nova_iguacu_df['id'] == acc_id]['tipo_veiculo'].unique()
    print(f"Tipos de Veículos Envolvidos: {', '.join(vehicle_types)}")


Informações Principais dos Acidentes Mais Graves:

Resumo do Acidente ID: 698382
Número de mortos: 0
Número de feridos (graves + leves): 4
Número de envolvidos (registros de pessoas/veículos): 4
BR: 116.0
KM: 183
Dia da Semana: domingo
Horário: 21:30:00
Causa do Acidente: Ausência de reação do condutor
Tipos de Veículos Envolvidos: Motocicleta


In [22]:
acidentes_analise = (
    nova_iguacu_df.groupby('id')
    .agg(
        total_mortos=('mortos', 'sum'),
        total_feridos_graves=('feridos_graves', 'sum'),
        total_feridos_leves=('feridos_leves', 'sum'),
        total_veiculos_envolvidos=('tipo_veiculo', 'count')
    )
    .reset_index()
)
acidentes_analise['total_feridos'] = (
    acidentes_analise['total_feridos_graves'] +
    acidentes_analise['total_feridos_leves']
)

print("\nInformações Principais dos Acidentes Mais Graves:")

acidente_mais_veiculos_id = acidentes_analise[acidentes_analise['total_veiculos_envolvidos'] == acidentes_analise['total_veiculos_envolvidos'].max()]['id'].iloc[0]

for acc_id in [int(acidente_mais_veiculos_id)]:
    print(f"\nResumo do Acidente ID: {acc_id}")

    agg_info = acidentes_analise[acidentes_analise['id'] == acc_id].iloc[0]
    print(f"Número de mortos: {int(agg_info['total_mortos'])}")
    print(f"Número de feridos (graves + leves): {int(agg_info['total_feridos'])}")
    print(f"Número de envolvidos (registros de pessoas/veículos): {int(agg_info['total_veiculos_envolvidos'])}")

    detail_info = nova_iguacu_df[nova_iguacu_df['id'] == acc_id].iloc[0]
    print(f"BR: {detail_info['br']}")
    print(f"KM: {detail_info['km']}")
    print(f"Dia da Semana: {detail_info['dia_semana']}")
    print(f"Horário: {detail_info['horario']}")
    print(f"Causa do Acidente: {detail_info['causa_acidente']}")

    vehicle_types = nova_iguacu_df[nova_iguacu_df['id'] == acc_id]['tipo_veiculo'].unique()
    print(f"Tipos de Veículos Envolvidos: {', '.join(vehicle_types)}")


Informações Principais dos Acidentes Mais Graves:

Resumo do Acidente ID: 672348
Número de mortos: 0
Número de feridos (graves + leves): 0
Número de envolvidos (registros de pessoas/veículos): 14
BR: 493.0
KM: 77
Dia da Semana: terça-feira
Horário: 08:20:00
Causa do Acidente: Manobra de mudança de faixa
Tipos de Veículos Envolvidos: Semireboque, Utilitário, Caminhão-trator, Desconhecido


### Investigação piores acidentes





**Resultado da pesquisa Acidente com maior número de feridos e Acidente com maior número de mortos**:

Após busca em portais de notícias (R7, Record, G1, O Dia, Jornal Destaque Baixada), redes sociais e comunicados oficiais, não foi possivel localizar qualquer matéria ou boletim de ocorrência público que descreva um acidente de moto em Nova Iguaçu em 15/06/2025 às 21h30 nem o atropelamento fatal de 23/04/2025.

É possível que o caso tenha sido apenas registrado em bancos de dados internos (como Detran/RJ) sem cobertura jornalística, ou publicado apenas em mídias restritas.

### Análise Geral Final

A análise dos dados de acidentes nas rodovias federais de Nova Iguaçu em 2025 mostrou que foram registrados 1.081 acidentes ao longo do ano.

A maior parte das ocorrências aconteceu na **BR-116**, que concentrou aproximadamente **72%** dos acidentes com **778 registros**. Em seguida aparecem a BR-465 com cerca de 17% e a BR-493 com aproximadamente 10%. Alguns trechos da BR-116, especialmente entre os quilômetros 181 e 187, apresentaram maior concentração de acidentes, indicando possíveis pontos críticos de trânsito.

Ao longo do ano, **novembro foi o mês com maior número de acidentes**, representando cerca de **12,2% do total de 132 registros**. Já outubro teve aproximadamente 6,7% e janeiro cerca de 6,8% , sendo os **meses com menor registro.**

No total, os acidentes resultaram em **17 mortes e 486 pessoas feridas**. Considerando os registros, cerca de 89,8% dos acidentes tiveram vítimas, enquanto aproximadamente 10,1% ocorreram sem vítimas.

Entre os tipos de acidentes, os **mais comuns foram colisão traseira**, responsável por cerca de **27,8% **dos casos , colisão lateral no mesmo sentido com aproximadamente **18,8%** e queda de ocupante de veículo, representando cerca de **18,3%**.

Em relação aos veículos envolvidos, **motocicletas em registros 414 vezes e automóveis 380**, sendo os tipos mais frequentes.

A análise também mostrou que a maior parte das pessoas envolvidas nos acidentes era do sexo masculino aproximadamente** 72%**, enquanto **17,8%** **eram do sexo feminino**.

Quanto ao momento das ocorrências, cerca de 60,3% dos acidentes aconteceram durante o dia e 30,9% durante a noite. Apesar de menos frequentes, **os acidentes noturnos concentraram aproximadamente 64,7% das mortes registradas**.

De forma geral, os dados indicam que os acidentes em Nova Iguaçu estão concentrados em rodovias de grande movimento, principalmente na BR-116, envolvendo com maior frequência motociclistas e condutores do sexo masculino, e podem estar relacionados a fatores como alto fluxo de veículos, comportamento de risco e atenção no trânsito.